Extending-OSVR/ConfiguringHDKViveTracking.md

```diff
@@ -0,0 +1,260 @@
+# Configuring OSVR HDK2 with Vive Puck Tracking
+
+This document outlines how to add room-scale positional tracking to HDK2 by mounting an HTC Vive tracking puck and using the Lighthouse base stations and tracking system.
+
+Hardware Requirements:
+OSVR HDK2 with mount
+Vive Tracking Puck
+Lighthouse base stations
+USB - microUSB cable
+
+
+Software Requirements:
+OSVR-Core build
+OSVR-Vive plugin build
+Optionally SteamVR
+
+### Why?
+There are a few reasons this document could be useful:
+- To serve as a working example of how to configure a new head/positional tracker using OSVR semantic paths.
+- To demonstrate how easily displays and tracking systems can be swapped quickly
+- To prove that the Vive tracking puck (with a wireless receiver) can be used for HDK headtracking in SteamVR
+
+### OSVR-Vive 
+[OSVR-Vive](https://github.com/OSVR/OSVR-Vive/) is the OSVR plugin for the family of Vive devices -- HMD, Controllers, and tracking puck. Wireless tracking of puck/controllers requires the Vive HMD or a wireless receiver. Follow the instructions in the Readme to install the plugin. Basically, download the latest binaries and copy **com_osvr_Vive.dll** to "/osvr-core-build/bin/osvr-plugins-0/".
```

Comment:
Which Readme? Can you link to it?

Put `com_osvr_Vive.dll` in backticks instead of bold. Same with destination directory.